### DISCUSSION TITLE
Time Consumption

### DISCUSSION THREAD

**Discussion by Ankkiyy** [lllyasviel/Fooocus#2779] [Q&A] [answered]
The generation of my images, on default settings, takes me about 50 minutes, which in turn ends up consuming a lot of energy. I am unsure if it is my network speed that is causing this issue or if it is the application itself. Is it the same for everyone? If not, is there any way to speed up this process without losing quality?

Specs : 

Model HP Gaming G15
Ram : 8 GB DDR4
GPU : 1650 GTX ( non Ti )
Storage : 512 NVMe
Processor : I5 11th Gen

**Comment by Rogala** [comment, marked as answer]
You have a very weak computer to work with artificial intelligence.

**Reply by 21BCA113** [reply]
My Friend is Having 35 seconds per image.

**Reply by mashb1t** [reply]
@Ankkiyy if there's a possibility to add another 8GB RAM please consider doing so as you're running Fooocus on the minimum required hardware specs.
